Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeInformation.ps1

```diff
@@ -154,6 +154,17 @@ function Get-ExchangeInformation {
                 Write-Verbose "Failed to run Test-ServiceHealth"
                 Invoke-CatchActions
             }
+
+            try {
+                $transportServiceResult = Get-TransportService -Identity $Server -ErrorAction Stop
+                $transportService = @{
+                    MaxPerDomainOutboundConnections = $transportServiceResult.MaxPerDomainOutboundConnections
+                    MessageRetryInterval            = $transportServiceResult.MessageRetryInterval
```

Comment:
Let's just store the whole thing, it should be small enough to manage, and this away it avoids cutting out other useful information.